### PR TITLE
[Feature1]First feature was implemented

### PR DESCRIPTION
**Pull request by milindr128**
following changes were made
1)added file1.txt
2)added file2.txt
3)added feature folder

### CODE REVIEW

File: feature/file1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+first feature
```
Comment (milindr12) [inline feature/file1.txt:1]: good milind128

Comment (milindr128) [reply feature/file1.txt:1]: hi from milindr 128 to milindr12

**Review by milindr12 — COMMENTED** [review]
good job milindr128

**Review by milindr12 — APPROVED** [review]
i am approving pull  request of milindr128

**Pull request merged**